### PR TITLE
sleuthkit 4.6.2

### PR DESCRIPTION
**Pull request by lembacon**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by DomT4** [comment]
Might be worth talking to @commitay about https://github.com/Homebrew/homebrew-core/pull/28238 & seeing if you can roll any of that into here.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
@rishwanth1995 Did you want to weigh in on this one? You mentioned something about the `jni` option in the other PR?

**Comment by rishwanth1995** [comment]
We want the jni option to be removed from the formula. We don't want our users to type in `with-jni` option to install the java bindings of sleuthkit, which in turn acts as a dependency for another software (Autopsy). Since, your analysis says users will most probably forget the extra option, can you remove the with-jni option? Thanks.

**Comment by DomT4** [comment]
To be clear are you saying you want it removed as in nobody can built with JNI support enabled, or are you saying you want JNI support to be mandatory?

**Comment by rishwanth1995** [comment]
We want the JNI support to be mandatory.

**Comment by DomT4** [comment]
Alright, that's fine with me, since IIRC it builds successfully with Java 10. Should allow us to finally update `autopsy` as well.

File: Formula/sleuthkit.rb [hunk 1]
```diff
@@ -11,37 +11,26 @@ class Sleuthkit < Formula
...
+  depends_on "libewf"
+  depends_on "libpq"
+  depends_on "sqlite"
 
   conflicts_with "irods", :because => "both install `ils`"
   conflicts_with "ffind",
     :because => "both install a 'ffind' executable."
 
   def install
-    ENV.append_to_cflags "-DNDEBUG" if build.without? "debug"
```
Comment (DomT4) [inline Formula/sleuthkit.rb]: Do we want to keep this line? We _usually_ don't build stuff with debug symbols/whatnot available by default.

Comment (lembacon) [reply Formula/sleuthkit.rb]: Agreed. Restored in https://github.com/Homebrew/homebrew-core/pull/30986/commits/b9168e22ccfb99f02508e9930c09ae7c503ca4e0.

**Comment by lembacon** [comment]
@BrewTestBot test this please

**Review by DomT4 — APPROVED** [review]

**Comment by DomT4** [comment]
Feel free to ship this whenever you're happy @lembacon 👍 .

**Pull request closed without merging**